src/main/scala/caio/std/CaioContext.scala

```diff
@@ -17,14 +17,14 @@ class CaioContext[C, V, L:Monoid] extends Context[Caio[C, V, L, *], C, L, Throwa
 
       def monadState: MonadState[FC, C2] = ???
 
-      def apply[A](c: C2)(f: FC[A]): Caio[C, V, L, A] =
+      def apply[A](c2: C2)(f: FC[A]): Caio[C, V, L, A] =
         CaioKleisli[C, V, L, A]{c =>
           IOResult {
-            f.eval(M.mix(c)).map {
+            f.eval(M.mix(c -> c2)).map {
```

Comment:
Could just be `f.eval(c2)`. Depends what needs to happen if both `C` and `C2` contain an `Int`. If we take the `int` from `C`, keep this. If we need the `Int` from `C2`, this line should be `f.eval(c2)`